java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.arrow.vector;
 
+import static java.util.Collections.emptyList;
```

Comment:
nit: would it be better to use a non-static import and use `emptyList`?

Reply:
happy to if that's the convention, yep :slightly_smiling_face:

Reply:
+1 for non-static

Reply:
updated :slightly_smiling_face: